src/algebra/hom/group.lean

```diff
@@ -1062,25 +1063,25 @@ left_inv_eq_right_inv (map_mul_eq_one f $ inv_mul_self x) $
   h.symm ▸ map_mul_eq_one g $ mul_inv_self x
 
 /-- Group homomorphisms preserve inverse. -/
-@[to_additive]
-protected theorem map_inv {G H} [group G] [group H] (f : G →* H) (g : G) : f g⁻¹ = (f g)⁻¹ :=
-map_inv f g
+@[to_additive] protected lemma map_inv [group α] [division_monoid β] (f : α →* β) :
```

Comment:
Rather than reflowing this line, lets add the missing docstring:
```suggestion
@[to_additive "Additive group homomorphisms preserve negation."]
protected lemma map_inv [group α] [division_monoid β] (f : α →* β) :
```